20-logging.py

```diff
@@ -17,25 +20,34 @@
 
 try:
     from lsst.rsp import IPythonHandler, forward_lsst_log
-
     customlogger = True
 except ImportError:
     pass  # Probably a container that doesn't have our new code
 
 # If the whole container is in debug mode, enable debug logging by default.
-#  Otherwise, set info as the default level.
-t_level = "INFO"
-level = logging.INFO
+# Otherwise, set warning as the default level; however, lsst logs should
+# be at info level. 
 debug = os.getenv("DEBUG")
-if debug:
-    t_level = "DEBUG"
-    level = logging.DEBUG
-# Set up WARNING and above as stderr, below that to stdout.
-handlers = [
-    logging.StreamHandler(level=logging.WARNING),
-    logging.StreamHandler(level=level, stream=sys.stdout),
-]
+handlers = []
 if customlogger:
-    forward_lsst_log(t_level)
+    # Forward anything at INFO or above, unless debug is set, in which case,
+    # forward DEBUG and above.
+    if debug
+        forward_lsst_log(logging.DEBUG)
```

Comment:
Did you modify this function to take an integer instead of a string?

Reply:
No, but the re-revision does.